testcases/unit/pintest.c

```diff
@@ -22,21 +22,12 @@ int main(int argc, char *argv[])
 	switch(getopt(argc, argv, "pn")) {
 	case 'p':
 		pin = pin_prompt(&buf, "Enter the PIN: ");
-#if defined(_AIX)
 		/* AIX prints an empty string instead of Linux-style (null) */
-		if (pin == NULL)
-			pin = "(null)";
-#endif
-		printf("pin: %s\n", pin);
+		printf("pin: %s\n", pin != NULL ? pin : "(null)");
```

Comment:
So this change basically reverts a change that you did in an earlier commit, right? I would usually prefer to update that earlier commit to not do the change at all, instead of changing it back here.